### ISSUE TITLE
merge command detects .k2 filetype but produces no output

### ISSUE THREAD

**Issue by zoey-rw** [cdiener/architeuthis#3]
When I run "merge" on Kraken2 output files, I get confirmation that the format was detected correctly, but the command completes immediately without generating an output file. I've tried with and without an out file path, and with wildcard expansion (*k2) as well as providing two full input file paths. The merge command is working for Bracken report files. Any ideas what might be going on?

Thanks!

```
architeuthis merge -o /projectnb/frpmars/soil_microbe_db/NEON_metagenome_classification/soil_microbe_db_merged_kraken.csv /projectnb/frpmars/soil_microbe_db/NEON_metagenome_classification/01_kraken_output/*.k2


2024/09/22 23:14:19 Detected format for files is 'kraken2'.
```

Conda info in case it's useful!
```
     active environment : struo2
    active env location : /projectnb/talbot-lab-data/zrwerbin/.conda/envs/struo2
            shell level : 1
       user config file : /usr3/graduate/zrwerbin/.condarc
 populated config files : /share/pkg.8/miniconda/24.5.0/install/.condarc
                          /usr3/graduate/zrwerbin/.condarc
          conda version : 24.5.0
    conda-build version : not installed
         python version : 3.12.4.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=cascadelake
                          __conda=24.5.0=0
                          __glibc=2.28=0
                          __linux=4.18.0=0
                          __unix=0=0
       base environment : /share/pkg.8/miniconda/24.5.0/install  (read only)
      conda av data dir : /share/pkg.8/miniconda/24.5.0/install/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://conda.anaconda.org/bioconda/linux-64
                          https://conda.anaconda.org/bioconda/noarch
                          https://repo.anaconda.com/pkgs/main/linux-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/linux-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /projectnb/talbot-lab-data/zrwerbin/.conda/pkgs
       envs directories : /projectnb/talbot-lab-data/zrwerbin/.conda/envs
                          /projectnb/talbot-lab-data/metabolic_models/scripts/metaGEM/envs
                          /usr3/graduate/zrwerbin/.conda/envs
                          /share/pkg.8/miniconda/24.5.0/install/envs
               platform : linux-64
             user-agent : conda/24.5.0 requests/2.32.3 CPython/3.12.4 Linux/4.18.0-553.16.1.el8_10.x86_64 almalinux/8.10 glibc/2.28 solver/libmamba conda-libmamba-solver/24.1.0
 libmambapy/1.5.8
                UID:GID : 283690:202078
             netrc file : /usr3/graduate/zrwerbin/.netrc
           offline mode : False
```

**Comment by cdiener** [comment]
I think I see what is going on. I will send in a fix.

**Comment by cdiener** [comment]
Okay, should be fixed in version 0.3.1 . I also added a test for this.
